episodes/chapter4.md

```diff
@@ -202,45 +143,25 @@ You will see a message:
 
 ![Collaboration details](fig/create_collab_03.png)
 
-### Reset API key for a node
-If you want to reset the API key for a node, you can do so by following these steps:
-
-1. Click on the `Nodes` tab in the administration page.
-2. Click on the tab of the node you want to reset the API key for in the list of nodes.
-
-![Reset API key for a node](fig/reset_api_key_01.png)
-
-3. Click on the `Reset API key` button.
-    * You will see a dialog box to ask you to download the new API key.
-
-![Download new API key](fig/reset_api_key_02.png)
-
-You will see a message:
-
-> API key download
->
-> Your API key has been reset. Please read your new key in the file that has been downloaded.
-
-At this point, put the new API key in your node configuration file and restart the node to connect to the server again.
 
 ::::::::::::::::::::::::::::::::::::: challenge
 
-## Challenge 3: Manage vantage6 project using the UI
+## Challenge 2: Manage vantage6 project using the UI
 
-Go the administration page in the vantage6 UI and try to create a new organization, a new user, and a new collaboration. Also, try to reset the API key for a node.
+Go to the administration page in the vantage6 UI, and try to do the following:
 
-1. Can you create a new organization, a new user, and a new collaboration?
-2. Can you reset the API key for a node?
-3. Which steps are challenging for you?
+1. Create a new organization
+2. Create a new user
+3. Create a new collaboration
 
-::::::::::::::::::::::::::::::::::::::::::::::::
+Which steps are challenging for you?
 
+::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: keypoints
 
-- Vantage6 entities are `Users`, `Organizations`, `Collaborations`, `Roles`, `Nodes`, and `Tasks`.
 - Vantage6 uses a permission system to control who can do what in the system.
 - Vantage6 has default roles like  `Root`, `Collaboration Admin`, `Organization Admin`, `Researcher`, and `Viewer`.
-- Vantage6 UI can be used to manage the entities of vantage6, like creating a new organization, a new user, a new collaboration, and resetting an api key for a node.
+- Vantage6 UI can be used to manage the entities of vantage6, like creating a new organization, a new user, or a new collaboration.
```

Comment:
We should mention somewhere that you can of course also delete resources (though we're not going to let them do it themselves as they will not have the permission because they could mess up the entire workshop - we could e.g. show it on the screen)

Reply:
The challenge 1 has mentioned that, and they also can see it in the figures 1 & 2.